common/types/activation.go

```diff
@@ -119,9 +126,24 @@ func (atxh *ActivationTxHeader) TargetEpoch() EpochID {
 	return atxh.PubLayerID.GetEpoch() + 1
 }
 
-// GetWeight of the atx.
+// GetWeight of the ATX. The total weight of the epoch is expected to fit in a uint64 and is
+// sum(atx.NumUnits * atx.TickCount for each ATX in a given epoch).
+// Space Units sizes are chosen such that NumUnits for all ATXs in an epoch is expected to be < 10^6.
```

Comment:
`10^6` for all ATXs is wrong. Using Chia as a reference, their historical maximum netspace was >36 EiB which is 151,000,000 space units of 256 GiB. So we should expect / be prepared for `10^9`.